tools/container/container.go

```diff
@@ -36,13 +36,14 @@ const (
 
 // Options defines the shared options for the CLIs to configure containers runtimes.
 type Options struct {
-	Config        string
-	Socket        string
-	RuntimeName   string
-	RuntimeDir    string
-	SetAsDefault  bool
-	RestartMode   string
-	HostRootMount string
+	Config           string
+	Socket           string
+	RuntimeName      string
+	RuntimeDir       string
+	SetAsDefault     bool
+	RestartMode      string
+	HostRootMount    string
+	RuntimeEnableCDI bool
```

Comment:
I don't agree with the `Runtime*` prefix here. `Runtime` in this context refers to the NVIDIA runtime. I would just use `EnableCDI` here.